README.md

```diff
@@ -1,2 +1,32 @@
-# terraform-is
-Terraform scripts for provision IS deployments on Azure cloud
+# Installation
+
+### Prerequisites
+
+* Install and set up [Packer](https://www.packer.io/) (>= v1.4.0 )
+* Install and set up [Terraform](https://www.terraform.io/) (>= v0.12.00 )
+
+
+### Instructions:
+
+1. Download the WSO2  Identity Server terraform resource.
+
+2. Build the Identity Server custom image. 
+
+   **Note:**  If you have an Azure subscription, update user variables in `centos-base.json`  file using your subscription credentials. The WSO2 Identity Server 5.10.0 distribution needs to download into the  `ansible-is/files/packs` directory. If you do not have a WSO2 subscription account, you can sign up for a free trial [here](https://wso2.com/free-trial-subscription). 
```

Comment:
@rasanjaya85 shall we just mention this as an active WSO2 Subscription? I suppose we no longer promote free trials and consider this as an active subscription (please see [example](https://github.com/wso2/docker-apim/tree/master/docker-compose/apim-with-analytics#prerequisites)).